.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/yaml-test-suite"]
+	path = test/yaml-test-suite
+	url = git@github.com:yaml/yaml-test-suite.git
```

Comment:
Let's vendor this rather than making it a git submodule

Reply:
Would you prefer to have the test suite inside a `/vendor` directory or keep it in the `/test` directory as it currently is?